README.md

```diff
@@ -27,82 +28,112 @@ Install `yarn` globally:
 npm install --global yarn
 ```
 
-## Set environment variables
+## Add sf-toolkit package
+
+Create `package.json` by running the following command and run through the prompts:
+
+```
+yarn init
+```
+
+Install `sf-toolkit` as a dependency of your project:
 
-You can either add the environment variables locally in the file so you only need to add them once, or define them directly in the terminal. 
+```
+yarn add https://github.com/silverfin/sf-toolkit.git
```

Comment:
We should eventually consider publishing an official npm package so they can add by doing something like `yarn add sf-toolkit` or `yarn add silverfin`

Reply:
Yes, indeed. `yarn` or `npm run` should be temporary solutions until we publish an npm package